### PR TITLE
feat(arc-488): only show audio thumb if audio is available

### PR DESCRIPTION
**Pull request by bertyhell**
If the objects have a thumbnail (which is the case when they have the BEZOEKERTOOL-CONTENT license), render them with the fake audio thumbnail:
image

if they do not have a thumbnail, then render the no-audio icon:
image

### CODE REVIEW

**Review by ian-emsens-shd — COMMENTED** [review]
Should this be in draft?

**Comment by bertyhell** [comment]
> Should this be in draft?

No, but you might need: https://github.com/viaacode/hetarchief-proxy/pull/188

**Comment by ian-emsens-shd** [comment]
Waiting on https://github.com/viaacode/hetarchief-proxy/pull/188

**Comment by bertyhell** [comment]
@ian-emsens-shd https://github.com/viaacode/hetarchief-proxy/pull/188 has been merged

File: src/modules/shared/components/MediaCard/MediaCard.tsx [hunk 1]
```diff
@@ -98,7 +98,10 @@ const MediaCard: FC<MediaCardProps> = ({
 	const renderHeader = () => {
 		switch (type) {
 			case 'audio':
-				return renderImage('/images/waveform.svg');
+				// Only render the waveform if the thumbnail is available
```
Comment (BavoVdg) [inline src/modules/shared/components/MediaCard/MediaCard.tsx:101]: Moet dit niet audio zijn ipv thumbnail?

**Comment by bertyhell** [comment]
> Ik zie geen waveform, maar de items hebben wel de licentie en een thumbnail url

@BavoVdg vreemd, die ene regel die de waveform rendered was weg. Het is nu fixed

**Pull request merged**